unit_test/test_spdm_responder/csr.c

```diff
@@ -961,6 +1090,84 @@ void libspdm_test_responder_csr_case10(void **state)
     free(m_libspdm_get_csr_request);
 }
 
+/**
+ * Test 11: receives a valid GET_CSR request message from Requester
+ * Expected Behavior: produces a valid CSR response message with alias_cert mode
+ **/
+void libspdm_test_responder_csr_case11(void **state)
+{
+    libspdm_return_t status;
+    libspdm_test_context_t *spdm_test_context;
+    libspdm_context_t *spdm_context;
+    size_t response_size;
+    uint8_t response[LIBSPDM_MAX_SPDM_MSG_SIZE];
+    spdm_csr_response_t *spdm_response;
+    spdm_get_csr_request_t *m_libspdm_get_csr_request;
+    uint8_t wrong_csr[LIBSPDM_MAX_CSR_SIZE];
+    bool result;
+    bool is_device_cert_model;
+
+    libspdm_zero_mem(wrong_csr, LIBSPDM_MAX_CSR_SIZE);
+
+    spdm_test_context = *state;
+    spdm_context = spdm_test_context->spdm_context;
+    spdm_test_context->case_id = 0xB;
+    spdm_context->connection_info.version = SPDM_MESSAGE_VERSION_12 <<
+                                            SPDM_VERSION_NUMBER_SHIFT_BIT;
+
+    spdm_context->connection_info.connection_state =
+        LIBSPDM_CONNECTION_STATE_NEGOTIATED;
+    spdm_context->local_context.capability.flags |=
+        SPDM_GET_CAPABILITIES_RESPONSE_FLAGS_CSR_CAP;
+    spdm_context->connection_info.algorithm.base_hash_algo =
+        m_libspdm_use_hash_algo;
+    spdm_context->connection_info.algorithm.base_asym_algo =
+        m_libspdm_use_asym_algo;
+
+    /*set alias cert mode*/
+    spdm_context->local_context.capability.flags |=
+        SPDM_GET_CAPABILITIES_RESPONSE_FLAGS_ALIAS_CERT_CAP;
+    is_device_cert_model = false;
+    if((spdm_context->local_context.capability.flags &
+        SPDM_GET_CAPABILITIES_RESPONSE_FLAGS_ALIAS_CERT_CAP) == 0) {
+        is_device_cert_model = true;
+    }

```

Comment:
here `libspdm_test_responder_csr_case11` is `with alias_cert mode`.
I think we already set the test scenario explicitly at the line 1128 and 1130, so remove the above line 1131 and 1134.

Reply:
Thanks. I have changed the code.